cmd/agent/app/agent.go

```diff
@@ -53,8 +53,10 @@ func NewAgentCommand(ctx context.Context) *cobra.Command {
 	karmadaConfig := karmadactl.NewKarmadaConfig(clientcmd.NewDefaultPathOptions())
 
 	cmd := &cobra.Command{
-		Use:  "karmada-agent",
-		Long: `The karmada agent runs the cluster registration agent`,
+		Use: "karmada-agent",
+		Long: `The karmada-agent runs the cluster registration agent on member clusters. 
+It will pull the user-submitted application and run it locally. Examples of controllers in karmada-agent
+are cluster-status-controller, execution-controller, work-status-controller and service-export-controller.`,
```

Comment:
```suggestion
		Long: `The karmada-agent is the agent of member clusters. It can register a specific cluster to the Karmada control
plane and sync manifests from the Karmada control plane to the member cluster. In addition, it also syncs the status of member 
cluster and manifests to the Karmada control plane.`,
```

Here is my suggestion. It's might not be a good idea to enumerate all the controllers here, because it's a little bit too specific, and might always forget to update when we introduce or deprecate controllers to/from it.